wmn-data.json

```diff
@@ -193,7 +193,7 @@
         "name" : "Airline_Pilot_Life",
         "uri_check" : "https://airlinepilot.life/u/{account}",
         "e_code" : 200,
-        "e_string" : "<title> Profile -",
+        "e_string" : "<title>  Profile -",

```

Comment:
for this site, we can use the `https://airlinepilot.life/u/{account}.json` for the `uri_check` and the `https://airlinepilot.life/u/{account}` for the `uri_pretty`. Then can pick one of the common parameters from the JSON like `primary_group_name` or `trust_level`. If you don't want to do this rework, I'm happy to.

Reply:
That'll work too. As will this fix.